core/src/main/scala/com/salesforce/op/filters/Summary.scala

```diff
@@ -64,3 +75,162 @@ case object Summary {
     }
   }
 }
+
+class TextSummary(textFormula: TextSummary => Int) extends Serializable {
```

Comment:
I'm quite confused here, is old Summary class still in use? or it's completely replaced by the new ones (text summary and numeric summary)

Reply:
Summary class seems no longer needed.